src/main/java/md/miller1995/Dealership/models/entities/CarEntity.java

```diff
@@ -15,7 +15,7 @@
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-public class Car {
+public class CarEntity {
     @Id
     @Column(name = "id")
```

Comment:
Don't we need/want an autoincrement strategy for `id`? (@GeneratedValue(strategy=GenerationType.AUTO))
https://stackoverflow.com/questions/2011528/hibernate-auto-increment-id

Reply:
- add annotation @GeneratedValue for field id in CarEntity class;